site/_layouts/doc.liquid

```diff
@@ -279,7 +279,7 @@
                   <ul>
                     <li {% if page.url == "/docs/v1/tech/account-management/" %}class="active"{% endif %}><a href="/docs/v1/tech/account-management/" >Overview</a></li>
                     <li {% if page.url == "/docs/v1/tech/account-management/updating-user-data/" %}class="active"{% endif %}><a href="/docs/v1/tech/account-management/updating-user-data/">Updating User Data & Password</a></li>
-                    <li {% if page.url == "/docs/v1/tech/account-management/totp-two-factor/" %}class="active"{% endif %}><a href="/docs/v1/tech/account-management/totp-two-factor/">Add TOTP Factor</a></li>
+                    <li {% if page.url == "/docs/v1/tech/account-management/authenticator-two-factor/" %}class="active"{% endif %}><a href="/docs/v1/tech/account-management/authenticator-two-factor/">Add Authenticator Factor</a></li>
```

Comment:
We should also add a redirect for google and anyone who may have bookmarked it.

* Can you please do so? Example here: https://raw.githubusercontent.com/FusionAuth/fusionauth-site/master/site/docs/v1/tech/troubleshooting.adoc (it's the `redirect_from` value in the front matter)
* Can you please add a note in the devdocs readme file about doing so?